### PR TITLE
Templates must be inline with the internal repo

### PR DESCRIPTION
**Pull request by ambiknai**

### CODE REVIEW

**Review by meghnasiingh — DISMISSED** [review]

File: deploy/kubernetes/driver/kubernetes/manifests/controller-server.yaml [hunk 1]
```diff
@@ -155,6 +160,29 @@ spec:
           volumeMounts:
             - name: socket-dir
               mountPath: /csi
+        - name: storage-secret-sidecar
```
Comment (arahamad) [inline deploy/kubernetes/driver/kubernetes/manifests/controller-server.yaml:163]: Opensource lib should not have sidecar related changes, can you remove all sidecar related reference

**Review by arahamad — CHANGES_REQUESTED** [review]
Can you do that changes

**Review by arahamad — APPROVED** [review]
LGTM

**Review by sameshai — APPROVED** [review]

**Review by meghnasiingh — APPROVED** [review]

**Review by arahamad — APPROVED** [review]
LGTM

**Pull request merged**